### DISCUSSION TITLE
mitmproxy socks5 error

### DISCUSSION THREAD

**Discussion by marvineichorstkpy** [mitmproxy/mitmproxy#6867] [General]
Version：mitmproxy  10.2.3
shell: mitmproxy --mode socks5 
error :Unsupported SOCKS5 request: b '\x05\x03\x00\x01\x00\x00\x00\x00\x00'
I found out that mitmproxy versions after 10.2.0 support udp. how is this going to work? Is it socks5 protocol support.

**Comment by mhils** [comment]
Our SOCKS proxy mode does not support UDP yet. I'd be happy for contributions, but it's probably not trivial to add. :) Probably most useful to do a Rust rewrite for the entire SOCKS proxy mode.
